server.go

```diff
@@ -592,6 +593,28 @@ func (s *Server) CreateClient(w http.ResponseWriter, r *http.Request, ps httprou
 	c := s.Config.GetUserConfig(user)
 	log.Debugf("user config: %#v", c)
 
+	if *maxNumberClientConfig > 0 {
+		if len(c.Clients) >= *maxNumberClientConfig {
+			log.Error(fmt.Errorf("user %q have too many configs", c.Name))
+
+			e := struct {
+				Error string
+			}{
+				Error: "Max number of configs: " + strconv.Itoa(*maxNumberClientConfig),
+			}
+
+			j, err := json.Marshal(e)
+			if err != nil {
+				log.Error(err)
+				return
+			}
+
+			w.WriteHeader(http.StatusTooManyRequests)
```

Comment:
Sorry for being a bit late to the party. 429 i.e. StatusTooManyRequests usually indicates the user has sent too many requests in a given amount of time however limiting the number of configs each user may have sounds more like a bad request. Would it make sense to change it to a 400 instead?

Otherwise it looks great!

Reply:
Yes, thought of that. But the request is not malformed or invalid.

Maybe 403 Forbidden is better suited?

"The server understood the request, but is refusing to fulfill it. Authorization will not help and the request SHOULD NOT be repeated." from https://www.w3.org/Protocols/rfc2616/rfc2616-sec10.html

Reply:
Have read some more rfcs and I agree that 400 might be "least bad" as it is more generic. Should we settle on 400?

Reply:
Sounds good @spetzreborn!

Reply:
Updated the PR, think all my insecurities are resolved :) Happy weekend!